azd-remote-hosting-templates/Azure.Mcp.Server/aca-managed-identity/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure MCP Server - ACA with Managed Identity
+
+Reference Azure Developer CLI (azd) template for deploying Azure MCP Server to Azure Container Apps with storage tools enabled.
+
+The MCP server storage tools uses the Container App's managed identity for outgoing authentication to Azure Storage.
+
+AI Foundry agents access the MCP server storage tools using the AI Foundry project's managed identity for incoming authentication.
+
+## Prerequisites
+
+- Azure subscription with **Owner** or **User Access Administrator** permissions
+- [Azure Developer CLI (azd)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
+
+## Quick Start
+
```

Comment:
add a note how this will be read-only and HTTP/s only and give them a link to more details on customizing their server startup flags